### ISSUE TITLE
Multiple Canvas

### ISSUE THREAD

**Issue by roholazandie** [p5py/p5#109]
I worked with p5js and there is this nice feature of multiple canvases that you can define extra canvases to paint without interfering with the main canvas. One can be found here:

https://editor.p5js.org/codingtrain/sketches/7RN7GFD-Y

As far as I know, there is a lack of this feature in p5 python version. Am I right or we have to wait to add this one?

**Comment by parsoyaarihant** [comment]
You are right @roholazandie . p5py is still a work in progress and we hope to add this feature in the future.
